powerauth-webflow-authentication-login-2fa/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/method/login2fa/controller/Login2FaInitController.java

```diff
@@ -0,0 +1,256 @@
+import io.getlime.security.powerauth.lib.dataadapter.model.response.UserDetailResponse;
+import io.getlime.security.powerauth.lib.nextstep.client.NextStepClient;
+import io.getlime.security.powerauth.lib.nextstep.model.entity.ApplicationContext;
+import io.getlime.security.powerauth.lib.nextstep.model.enumeration.AuthMethod;
+import io.getlime.security.powerauth.lib.nextstep.model.enumeration.AuthStepResult;
+import io.getlime.security.powerauth.lib.nextstep.model.enumeration.OperationCancelReason;
+import io.getlime.security.powerauth.lib.nextstep.model.exception.NextStepServiceException;
+import io.getlime.security.powerauth.lib.nextstep.model.response.GetOperationDetailResponse;
+import io.getlime.security.powerauth.lib.nextstep.model.response.GetOrganizationDetailResponse;
+import io.getlime.security.powerauth.lib.nextstep.model.response.GetOrganizationListResponse;
+import io.getlime.security.powerauth.lib.webflow.authentication.base.AuthStepResponse;
+import io.getlime.security.powerauth.lib.webflow.authentication.controller.AuthMethodController;
+import io.getlime.security.powerauth.lib.webflow.authentication.exception.AuthStepException;
+import io.getlime.security.powerauth.lib.webflow.authentication.exception.CommunicationFailedException;
+import io.getlime.security.powerauth.lib.webflow.authentication.method.login2fa.model.request.Login2FaInitRequest;
+import io.getlime.security.powerauth.lib.webflow.authentication.method.login2fa.model.request.PrepareLoginFormDataRequest;
+import io.getlime.security.powerauth.lib.webflow.authentication.method.login2fa.model.response.Login2FaInitResponse;
+import io.getlime.security.powerauth.lib.webflow.authentication.method.login2fa.model.response.PrepareLoginFormDataResponse;
+import io.getlime.security.powerauth.lib.webflow.authentication.model.HttpSessionAttributeNames;
+import io.getlime.security.powerauth.lib.webflow.authentication.model.OrganizationDetail;
+import io.getlime.security.powerauth.lib.webflow.authentication.model.converter.FormDataConverter;
+import io.getlime.security.powerauth.lib.webflow.authentication.model.converter.OrganizationConverter;
+import io.getlime.security.powerauth.lib.webflow.authentication.service.AuthMethodQueryService;
+import io.getlime.security.powerauth.lib.webflow.authentication.service.AuthenticationManagementService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpSession;
+import java.util.List;
+
+
+/**
+ * Controller for initialization of 2FA login.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@RestController
+@RequestMapping(value = "/api/auth/login-2fa/init")
```

Comment:
Maybe this should be just `/api/auth/login-2fa`?

Reply:
Yes, it's a left over from time when I thought we would have multiple endpoints in this method for init, sms, token, etc. I will fix it.